server/routes/handlers/projects.js

```diff
@@ -1,7 +1,8 @@
+const json2csv = require('json2csv');
 const projectController = require('../../controllers/projects');
 const userController = require('../../controllers/users');
 const eventController = require('../../controllers/events');
-const json2csv = require('json2csv');
+const CSVHeaderSerializer = require('../../models/projectCSVSerializer');
```

Comment:
I'm gonna be super picky here... `csvHeaderSerializer` since it's a function, not a class.

Also, the file isn't an overall serializer like the name suggests. Should be renamed appropriately.

Reply:
Open to suggestions for name :)

Reply:
`projectsCSVHeaderGenerator`? Can't really think of anything beyond that really... Unless you were to move everything CSV related to one class and just call it projectCSVSerializer then...